criu/cr-restore.c

```diff
@@ -3128,6 +3128,24 @@ static void rst_reloc_creds(struct thread_restore_args *thread_args,
 	thread_args->creds_args = args;
 }
 
+static bool groups_are_okay(gid_t* groups, int n_groups) {
```

Comment:
I think a better name would be `groups_match()` as it conveys that we compare the supplied groups with existing ones.
Another nit, the opening brace should be on a separate line.